larray/inout/string.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import absolute_import, print_function
+
+import numpy as np
+from pandas import DataFrame
+from larray.util.misc import StringIO, deprecate_kwarg
+from larray.inout.utils import _get_index_col
+from larray.inout.pandas import df_aslarray
+from larray.inout.csv import read_csv
+
+
+__all__ = ['from_lists', 'from_string']
+
+
+@deprecate_kwarg('nb_index', 'nb_axes', arg_converter=lambda x: x + 1)
+def from_lists(data, nb_axes=None, index_col=None, fill_value=np.nan, sort_rows=False, sort_columns=False, wide=True):
```

Comment:
does not really belong in a string module. Maybe rename the module to misc?

Reply:
Sure